Assets/Scripts/GrindingGame/DecoratorCode/KeyPromptDecorator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KeyPromptDecorator : IKeyPrompt
+{
+    private IKeyPrompt _KeyPrompt;
```

Comment:
Burke wants us to use the same variable naming conventions throughout so it's probably better to change this to something like "keyPrompt" since we haven't use "_Name" style conventions anywhere else

Reply:
Also add comments at top of classes and throughout all new code.